DashboardTests/StatusPageTest.cs

```diff
@@ -0,0 +1,39 @@
+using Avalonia.Controls;
+using Avalonia.Headless.NUnit;
+using Dashboard.Models;
+using Dashboard.ViewModels;
+using Dashboard.Views;
+using Moq;
+
+namespace DashboardTests;
+
+[TestFixture]
+public class OverviewPageTest
+{
+    [SetUp]
+    public void SetUp()
+    {
+        _dataStore = new Mock<IDataStore>();
+    }
+
+    private Mock<IDataStore> _dataStore;
+
+    [AvaloniaTest]
+    public void TestThatStatusPageRendersCorrectly()
+    {
+        var window = new StatusView
+        {
+            DataContext = new StatusViewModel(_dataStore.Object)
+        };
+
+        Assert.IsNotNull(window);
+
+        // Make sure there is a icon for RES
+        var resIcon = window.FindControl<Image>("ResIcon");
+        Assert.IsNotNull(resIcon);
+
+        Assert.Pass("Status page rendered correctly");
```

Comment:
I don't think this line is required, u can use Assert.Pass to early exit tests if a condition is met, but we are just finishing which is an implied success case idk

Reply:
Oki got it thanks!